src/guiapi/src/window_menu.cpp

```diff
@@ -271,7 +275,7 @@ void window_menu_t::unconditionalDraw() {
 
 void window_menu_t::redrawWholeMenu() {
     const int item_height = font->h + padding.top + padding.bottom;
-    const size_t visible_available = rect.Height() / item_height;
+    const size_t visible_available = rect.Height() / (item_height + 1);
```

Comment:
what is this "+1"?

Reply:
Fixed in force push.

Reply:
I still don't see the "1" being changed into some meaningful constant